<?xml version="1.0"?>
<launch>

	<arg name="working_dir" default="/tmp" />

	<param name="/rosplan/domain_path" value="$(find rosplan_config)/planner/domain.pddl" />
	<!-- directory for generated files -->
	<param name="/rosplan/data_path" value="$(arg working_dir)/" />
	<param name="/rosplan/problem_path" value="$(arg working_dir)/problem.pddl" />
	<param name="/rosplan/strl_file_path" value="$(arg working_dir)/plan.strl" />

	<!-- to run the planner -->
	<param name="/rosplan/planner_command" value="timeout 10 $(find rosplan_config)/planner/bin/popf -n DOMAIN PROBLEM" />
	<param name="/rosplan/generate_default_problem" value="true" />
	<param name="/rosplan/max_dispatch_attempts" value="2" />

	<!-- planning system -->
	<node name="rosplan_planning_system" pkg="rosplan_planning_system" type="planner" respawn="false" output="screen">
	</node>

</launch>
